.test-infra/jenkins/job_PerformanceTests_HadoopInputFormat.groovy

```diff
@@ -39,7 +39,9 @@ job(jobName) {
             tempRoot       : 'gs://temp-storage-for-perf-tests',
             project        : 'apache-beam-testing',
             postgresPort   : '5432',
-            numberOfRecords: '600000'
+            numberOfRecords: '600000',
+            bigQueryDataset: 'beam_performance',
+            bigQueryTable  : 'test_hadoopinputformat_it'
```

Comment:
There seemed to be a clear convention for those names: "<testname>_results" eg. "textioit_results". Can we stay consistent here too?

Reply:
Sure, done.